File: app/src/main/java/com/lewis/clean/model/AppInfo.java
package com.lewis.clean.model;


import android.graphics.drawable.Drawable;

public class AppInfo {
    private int id;
    private boolean isFilterProcess;
    private String packageName;
    private Drawable icon;
    private String name;
    private boolean isSystemProcess;
    private int memorySize;

    public AppInfo() {
    }

    public AppInfo(String packageName) {
        this.packageName = packageName;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setIsFilterProcess(boolean isFilterProcess) {
        this.isFilterProcess = isFilterProcess;
    }

    public void setPackageName(String packageName) {
        this.packageName = packageName;
    }

    public void setIcon(Drawable icon) {
        this.icon = icon;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setIsSystemProcess(boolean isSystemProcess) {
        this.isSystemProcess = isSystemProcess;
    }

    public void setMemorySize(int memorySize) {
        this.memorySize = memorySize;
    }

    public boolean getIsSystemProcess() {
        return isSystemProcess;
    }

    public boolean getIsFilterProcess() {
        return isFilterProcess;
    }

    public String getPackageName() {
        return packageName;
    }

    public int getMemorySize() {
        return memorySize;
    }
}

File: app/src/main/java/com/lewis/clear/model/AppInfo.kt
package com.lewis.clear.model


import android.graphics.drawable.Drawable

class AppInfo {
    private var id: Int = 0
    var isFilterProcess: Boolean = false
    var packageName: String? = null
    private var icon: Drawable? = null
    private var name: String? = null
    var isSystemProcess: Boolean = false
    var memorySize: Int = 0

    constructor() {}

    constructor(packageName: String) {
        this.packageName = packageName
    }

    fun setId(id: Int) {
        this.id = id
    }

    fun setIcon(icon: Drawable) {
        this.icon = icon
    }

    fun setName(name: String) {
        this.name = name
    }
}
